docs/specs/IDE-execution.md

```diff
@@ -198,6 +162,51 @@ The properties of the IDE endpoint information document are:
 | --- | --------- | --- |
 | `protocols_supported` | List of protocols supported by the IDE endpoint. See [protocol versioning](#protocol-versioning) for more information. | `string[]` |
 
+
+## Launch configurations (run session requests)
+
+The run session creation request contains one or more launch configurations for the session. The following launch configuration types are well-known:
+
+### Project launch configuration (type: `project`)
+
+Project launch configuration contains details for launching programs that have project files compatible with Visual Studio IDE.
+
+**Project launch configuration properties**
+
+| Property | Description | Required? |
+| --- | --------- | --- |
+| `type` | Launch configuration type indicator; must be `project`. | Required |
+| `project_path` | Path to the project file for the program that is being launched. | Required |
+| `mode` | Specifies the launch mode. Currently supported modes are `Debug` (run the project under the debugger) and `NoDebug` (run the project without debugging). | Optional, defaults to `Debug`. |
+| `launch_profile` | The name of the launch profile to be used for project execution. See below for more details on how the launch profile should be processed. | Optional |
+| `disable_launch_profile` | If set to `true`, the project will be launched without a launch profile and the value of "launch_profile" parameter is disregarded. | Optional |
```

Comment:
for .NET projects, Is `disable_launch_profile` always be false?

Reply:
No, see how `ExcludeLaunchProfileAnnotation` can be used to disable the launch profile, for example https://github.com/dotnet/aspire/blob/main/src/Aspire.Hosting/Dcp/DcpExecutor.cs#L957 and https://github.com/dotnet/aspire/blob/main/tests/Aspire.Hosting.Tests/ProjectResourceTests.cs#L289